modules/installation-minimum-resource-requirements.adoc

```diff
@@ -245,17 +245,6 @@ ifdef::vsphere[]
 2. As with all user-provisioned installations, if you choose to use {op-system-base} compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks. Use of {op-system-base} 7 compute machines is deprecated and has been removed in {product-title} 4.10 and later.
 endif::vsphere[]
 --
-[NOTE]
-====
-As of {product-title} version 4.13, RHCOS is based on RHEL version 9.2, which updates the micro-architecture requirements. The following list contains the minimum instruction set architectures (ISA) that each architecture requires:
```

Comment:
Do we really need to remove this part? Because this doc is for fresh installation, I think this section can guide customers to choose a proper CPU architecture. WDYT?

Reply:
Of course! Sorry about that :) I'll add the other links ASAP!